DOCS.md

```diff
@@ -172,12 +172,13 @@ This posts weather information of a given location.
 
 __Usage__
 
-`@weather <zipcode | location name>`
+`@weather <zipcode | location name> > <degree type>`
 
 __Arguments__
 
 * `zipcode`: A five digit US zipcode.
 * `location name`: A letter and space only location name, or city name and state separated by a comma and a space.
+* `degree type`: A degree type to display the weather, the parameter can be 'C' for Celsius or 'F' for Fahrenheit. The default type is Fahrenheit.
```

Comment:
@renxinhe Why can't we just poll for the a space and a C or an F? Then the old code can still be reused, and we don't have to introduce bash to non-programmers.